### PR TITLE
magento/devdocs#: GraphQl. API functional tests. Add info about tests location

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds information about GraphQl API functional tests location.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/functional-testing.html

## Links to Magento source code

-  [dev/tests/api-functional/testsuite/Magento/GraphQl/](https://github.com/magento/magento2/tree/2.3.3/dev/tests/api-functional/testsuite/Magento/GraphQl)

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/graphql/functional-testing.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Magento provides API functional tests that can verify extension points in GraphQ
 
 ## Creating a new GraphQL functional test
 
-All GraphQL functional tests should inherit from the generic test case `Magento\TestFramework\TestCase\GraphQlAbstract`. It defines the `graphQlQuery()` method, which should be used to perform Web API calls from tests.
+All GraphQL functional tests should locate in `dev/tests/api-functional/testsuite/Magento/GraphQl/` folder and inherit from the generic test case `Magento\TestFramework\TestCase\GraphQlAbstract`. It defines the `graphQlQuery()` method, which should be used to perform Web API calls from tests.
```
Comment (rogyar) [inline src/guides/v2.3/graphql/functional-testing.md]:
```suggestion
All GraphQL functional tests be located in `dev/tests/api-functional/testsuite/Magento/GraphQl/` folder and inherit from the generic test case `Magento\TestFramework\TestCase\GraphQlAbstract`. It defines the `graphQlQuery()` and `graphQlMutation()` methods, which should be used to perform Web API calls from tests.
```

Comment (atwixfirster) [reply src/guides/v2.3/graphql/functional-testing.md]: **fixed**

Thanks , @rogyar

Comment (rogyar) [reply src/guides/v2.3/graphql/functional-testing.md]: Thank you! Please, take a look at my suggestion above, there's information about `graphQlMutation()` method. I believe we should mention it as well.

Comment (keharper) [reply src/guides/v2.3/graphql/functional-testing.md]: Fixed.

The `master` branch is locked for now, until 2.3.4 published tomorrow. I'll merge the PR at that time.

Comment (atwixfirster) [reply src/guides/v2.3/graphql/functional-testing.md]: Thank you, @keharper !

I've squashed commits. So, now PR is ready for merge ;)

**Pull request merged**